.goreleaser.yaml

```diff
@@ -64,16 +64,16 @@ checksum:
   name_template: 'checksums.txt'
 
 signs:
-  - artifacts: all
+  - cmd: cosign
+    stdin: "{{ .Env.COSIGN_PASSWORD }}"
```

Comment:
```suggestion
```

`cosign` by default reads the password from this exactly environment variable so you don't need to provide it in the stdin.